ZipBuilder/Package.swift

```diff
@@ -38,11 +42,26 @@ let package = Package(
     ),
     .target(
       name: "ZipBuilder",
-      dependencies: ["ArgumentParser", "ManifestReader"]
+      dependencies: ["ArgumentParser", "ManifestReader", "ShellUtils"]
     ),
     .target(
       name: "ManifestReader",
       dependencies: ["SwiftProtobuf"]
     ),
+    .target(name: "ShellUtils"),
```

Comment:
Nit: I think we're slowly moving towards the `snake_case` style since it seems to be used and expected within SwiftPM so this should be `shell-utils`.

Reply:
It looks to me that the snake case names are good for command line targets to make it command line friendly (though even here I'm not sure if we should use snake case for products only and camel case for targets), but look a bit strange for libraries and internal dependencies. 

I don't feel as an expect here so I'm open for discussion. In either case I'll need to update either `sem-versions` or `ShellUtils` for consistency.